src/jit/lsraarm.cpp

```diff
@@ -727,6 +727,43 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
             info->internalIntCount = 1;
             break;
 
+        case GT_COPY:
+        case GT_PUTARG_REG:
+            // TODO-ARM-Cleanup : the logic was copied from general case,
+            // Only the exceptional case was added - when src count should be 2.
+            info->dstCount = tree->IsValue() ? 1 : 0;
+            if (kind & (GTK_CONST | GTK_LEAF))
```

Comment:
Both of these are known to be `GTK_UNOP`, so this is unnecessary (`GTK_SMPOP` includes `GTK_UNOP`).  I would suggest adding a method to GenTreeMultiRegOp such as `GetRegCount()`, and then you could simply do:
```
info->srcCount = tree->AsGenTreeMultiRegOp()->GetRegCount();
```
You could also assert based on type that `info->srcCount` has the right value.